bigframes/functions/remote_function_template.py

```diff
@@ -65,6 +102,7 @@ def get_pd_series(row):
         "Int64": int,
         "Float64": float,
         "string": str,
+        "binary[pyarrow]": base64.b64decode,
```

Comment:
This dictionary worries me. It seems pretty brittle to rely on the string representations of pandas dtypes. We should revisit this in b/345222844